### PR TITLE
Bar stool no longer becomes a normal stool and are now constructables

### LINKED ISSUE #18374: when you pick up and place a bar stool it turns into a regular stool
reeee

### PR DESCRIPTION
**Pull request by lzimann**
Fixes #18374 (this won't count for a fix token since the issue was caused by me)
My bad. Also adds sprites for toppled bar stools(same sprite but toppled)
Also makes them constructable with 1 sheet of metal.
:cl: Name: Lzimann
rscadd: Bar stools are now constructable with metal!
/:cl:
fix token #18372

### CODE REVIEW

File: code/game/objects/items/stacks/sheets/sheet_types.dm [hunk 1]
```diff
@@ -13,6 +13,7 @@
  */
 var/global/list/datum/stack_recipe/metal_recipes = list ( \
 	new/datum/stack_recipe("stool", /obj/structure/chair/stool, one_per_turf = 1, on_floor = 1), \
+	new/datum/stack_recipe("bar stool", /obj/structure/chair/stool/bar, one_per_turf = 1, on_floor = 1), \
```
Comment (pudl) [inline code/game/objects/items/stacks/sheets/sheet_types.dm]: did you test this

Comment (lzimann) [inline code/game/objects/items/stacks/sheets/sheet_types.dm]: yes

Comment (pudl) [inline code/game/objects/items/stacks/sheets/sheet_types.dm]: no you didnt

Comment (lzimann) [inline code/game/objects/items/stacks/sheets/sheet_types.dm]: if you are going to stay stuff for the sake of bothering me, just fuck off, please.

Comment (pudl) [inline code/game/objects/items/stacks/sheets/sheet_types.dm]: alright man hope you learn your lesson when this gets merged

Comment (optimumtact) [inline code/game/objects/items/stacks/sheets/sheet_types.dm]: @pudl please explain what you mean, there's no need to be mysterious or ask people riddles

Comment (lzimann) [inline code/game/objects/items/stacks/sheets/sheet_types.dm]: aaa
if this is not testing, well, I don't know what is.

Comment (pudl) [inline code/game/objects/items/stacks/sheets/sheet_types.dm]: alright i was worried that the addition of the bar stool would mess up the list/make it too long, just wanted to be sure it wouldnt happen

**Pull request merged**